### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#970] [Galaxy Trivia] [answered]
> It's not the 📏 of the 🐩 in the ⚔️, but ...

**Bonus:** Name the author.

**Comment by peterbaccaro** [comment, marked as answer]
The expression is as follows:

**It's not the size of the dog in the fight, its the size of the fight in the dog.**

The exact origin of the phrase is uncertain, and it has been attributed to different sources over time. However, it is often attributed to **Mark Twain**, although there is no concrete evidence linking it directly to him.

The phrase is an idiomatic expression that emphasizes the importance of determination, courage, and tenacity over physical size or strength. It suggests that an individual's inner strength, willpower, and resilience are more influential in determining their success or ability to overcome challenges than their outward attributes or disadvantages.

This phrase is often used to inspire and encourage individuals to believe in themselves and their abilities, regardless of any perceived limitations or obstacles they may face. It highlights the idea that one's mindset, attitude, and determination can make a significant difference in achieving goals or overcoming adversity.
